### PR TITLE
Improve troubleshooting following issues

### PR DESCRIPTION
**Pull request by fcollonval**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -28,12 +28,45 @@ jupyter lab build
 
 ### Troubleshooting
 
-- When you run JupyterLab, if you can see the Git sidepanel UI but you cannot get it to work, you may need to explicitly enable the serverextension by running:
+Before looking in the following list, be sure the server extension and the frontend extension have the same version. For that you can execute the following commands:
```
Comment (kgryte) [inline README.md]:
```suggestion
Before consulting the following list, be sure the server extension and the frontend extension have the same version by executing the following commands:
```

File: README.md [hunk 2]
```diff
@@ -28,12 +28,45 @@ jupyter lab build
...
 
 ```bash
-jupyter serverextension enable --py jupyterlab_git
+jupyter serverextension list
+jupyter labextension list
 ```
 
+- The Git panel does not recognized you are in a Git repository
```
Comment (kgryte) [inline README.md]:
```suggestion
- **Issue**: the Git panel does not recognize that you are in a Git repository.
```

File: README.md [hunk 3]
```diff
@@ -28,12 +28,45 @@ jupyter lab build
...
+
+  Possible fixes:
+  
+  - Be sure to be in a Git repository in the filebrowser tab
+  
+  - Check the server log, if you see a warning with a 404 code similar to:
```
Comment (kgryte) [inline README.md]:
```suggestion
  - Check the server log. If you see a warning with a 404 code similar to:  
```

File: README.md [hunk 4]
```diff
@@ -28,12 +28,45 @@ jupyter lab build
...
+  
+  - Check the server log, if you see a warning with a 404 code similar to:  
+    `[W 00:27:41.800 LabApp] 404 GET /git/server_root?1576081660665`
+    
+    Explicitly enable the server extension by running:
+    ```bash
+    jupyter serverextension enable --py jupyterlab_git
+    ```
+    
+  - If you are using JupyterHub or some other technologies requiring to set an initialization script including the jupyterlab-git extension, be sure to install the frontend and the server extension before launching JupyterLab.
```
Comment (kgryte) [inline README.md]:
```suggestion
  - If you are using JupyterHub or some other technologies requiring an initialization script which includes the jupyterlab-git extension, be sure to install both the frontend and the server extension **before** launching JupyterLab.
```

File: README.md [hunk 5]
```diff
@@ -28,12 +28,45 @@ jupyter lab build
...
+  
+- The Git panel is not visible
```
Comment (kgryte) [inline README.md]:
```suggestion
- **Issue**: the Git panel is not visible.
```

**Comment by fcollonval** [comment]
Thanks @kgryte for the review and corrections.

**Pull request merged**